eox_tagging/models.py

```diff
@@ -23,4 +23,4 @@ def __str__(self):
         Get a string representation of this model instance.
         """
         # TODO: return a string appropriate for the data fields
-        return '<Tag, ID: {}>'.format(self.id)
+        return str(self)
```

Comment:
Esto no va a llamarse recursivamente?

Reply:
IDK, probably. I'll change it just in case